js/Core/Chart/Chart.js

```diff
@@ -2449,7 +2449,12 @@ var Chart = /** @class */ (function () {
         var chart = this, lang = defaultOptions.lang, btnOptions = chart.options.chart.resetZoomButton, theme = btnOptions.theme, states = theme.states, alignTo = (btnOptions.relativeTo === 'chart' ||
             btnOptions.relativeTo === 'spaceBox' ?
             null :
-            'plotBox');
+            {
+                x: chart.plotLeft,
+                y: chart.plotTop,
+                width: chart.chartWidth - chart.plotLeft - (chart.marginRight || 0),
+                height: chart.chartHeight - chart.plotTop - (chart.marginBottom || 0)
+            });
```

Comment:
So with the `scrollablePlotArea` module we first mutate `chart.plotBox`...

https://github.com/highcharts/highcharts/blob/5553fad1efb18cb65c279c0d7517bd3ac8633db8/ts/Extensions/ScrollablePlotArea.ts#L157-L158

... then we need to unmutate it for this fix. I feel like we need to find a smarter way. Maybe we should store a copy of the original `plotBox` as `chart.scrollablePlotBox`? Or maybe more intiitively, the other way around, avoid mutating the `plotBox` and create a new `scrollablePlotBox`. So wherever we need to access the full width, access `(chart.scrollablePlotBox || chart.plotBox).width`. But first see in what cases the extended width is used.